README.md

```diff
@@ -10,3 +10,24 @@ This repository contains versioned TextField suggestion data for eq-questionnair
 | national-identities.json | List of national identities |
 | passport-countries.json | List of countries for passport questions |
 | religions.json | List of religions |
+
+
+Source data files are provided by the business as multiple column csv files. These can be manually added/updated in this repository at ./source-data
+
+- source files to be provided by the business named for the above datasets
+
+- files to be UTF-8 csv
+
+- any values including commas should be double quoted
+
+- header row to contain comma separated language_codes (i.e `en_gb`, `cy`, `ga`, `eo`)
+
+- data rows to contain comma separated translated lookup values (as to be presented in the lookup lists)
+
+- null values for a given language item can be provided by a non-spaced commas. This will result in that term not being presented as a suggestion for that language
+
+json files can be generated from the source csv files using `./scripts/convert_csv_to_json.py`
```

Comment:
```suggestion
json files can be generated from the source csv files using `pipenv run python scripts/convert_csv_to_json.py`
```
`/scripts/convert_csv_to_json.py` only works if you are in `pipenv shell`, the suggestion works if you are in shell or not. Alternatively, you can document that you need to have `pipenv shell` activated.

Reply:
Yes, good point.